crates/core/src/factories/core_fetch/test/outgoing_request_queue.rs

```diff
@@ -106,67 +103,10 @@ fn make_mock_transport(
     Arc::new(mock_transport)
 }
 
-#[tokio::test(flavor = "multi_thread")]
-async fn outgoing_request_queue() {
```

Comment:
Without the re-insertion logic, this test case is covered in the following happy path test.